pkg/gui.py

```
@@ -190,152 +213,132 @@ def check_status(self, thread):
-            # widget.after(6000, widget.destroy())
-            # widget.after(self.alive_time, lambda: widget.withdraw())
-            # widgets.append(widget)
-            print("--Displayed pop_always_on_top--")
-
-    def popup(self):
-        '''
-        pops a message overlay on the screen if one exists in the queue
-        '''
-        if time.now() > self.since_last_popup + timedelta(0, self.alive_time/1000):
-            #
+        if time.now() > self.since_last_popup + timedelta(seconds=self.alive_time / 1000):
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)